docs/src/docs/dates/date-range-picker.mdx

```diff
@@ -18,10 +18,16 @@ import { DateRangePickerDemos } from '@demos/dates';
 
 <Demo data={DateRangePickerDemos.usage} />
 
+## Multiple months
+
+You can show two months next to each other by setting `amountOfMonths`:
+
+<Demo data={DateRangePickerDemos.multiMonth} />
+
 ## DatePicker props
 
 DateRangePicker component uses the same props as [DatePicker](/dates/date-picker/) component.
-The only difference is in `value` and `onChange` props shown in previous example.
+The only difference is in `value`, `onChange` and `withMultipleMonths` props shown in previous examples.
```

Comment:
`withMultipleMonths`? Shouldn't it be `amountOfMonths`?

Reply:
You're right. Changed it to `amountOfMonths`